scripts/auto_fill_manifest.py

```
@@ -0,0 +1,342 @@
+            continue
+        placeholders.append(ph)
+
+    labels: list[Label] = []
+    for tag in ("svg:text", "svg:tspan"):
+        for elem in root.xpath(f'.//{tag}[@id]', namespaces=SVG_NS):
+            text = (elem.text or "").strip()
+            if not text:
+                # skip unlabeled <tspan> wrappers
+                continue
+            try:
+                x_raw = elem.get("x"); y_raw = elem.get("y")
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)